src/flow/src/expr/relation/accum.rs

```diff
@@ -157,6 +170,21 @@ pub struct SimpleNumber {
     non_nulls: Diff,
 }
 
+impl SimpleNumber {
+    pub fn try_from_iter<I>(iter: &mut I) -> Result<Self, EvalError>
```

Comment:
It would be great if you could describe the expected number and types of values in the iter.